docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,262 @@
+
+The third section of the `values.yaml` file allows one to create a list of Inventory and Transport Spaces (ITSes). By default, this list is empty and no ITS will be created by the chart. A list of ITSes can be specified using the following format:
+
+```yaml
+ITSes: # all the CPs in this list will execute the its.yaml PCH
+  - name: <its1>          # mandatory name of the control plane
+    type: <vlcuster|host> # optional type of control plane host or vcluster (default to vcluster, if not specified)
+  - name: <its2>          # mandatory name of the control plane
+    type: <vlcuster|host> # optional type of control plane host or vcluster (default to vcluster, if not specified)
+  ...
+```
+
+where `name` must specify a name unique among all the control planes in that KubeFlex deployment and the optional `type` can be either vlcuster (default) or host, see [here](https://github.com/kubestellar/kubeflex/blob/main/docs/users.md) for more information.
+
+The fourth section of the `values.yaml` file allows one to create a list of Workload Description Spaces (WDSes). By default, this list is empty and no WDS will be created by the chart. A list of WDSes can be specified using the following format:
+
+```yaml
+WDSes: # all the CPs in this list will execute the wds.yaml PCH
+  - name: <wds1>     # mandatory name of the control plane
+    type: <host|k8s> # optional type of control plane host or k8s (default to k8s, if not specified)
+    APIGroups: ""    # optional string holding a comma-separated list of APIGroups
+    ITSName: <its1>  # optional name of the ITS control plane, this MUST be specified if more than one ITS exists at the moment the WDS PCH starts
+  - name: <wds2>     # mandatory name of the control plane
+    type: <host|k8s> # optional type of control plane host or k8s (default to k8s, if not specified)
+    APIGroups: ""    # optional string holding a comma-separated list of APIGroups
+    ITSName: <its2>  # optional name of the ITS control plane, this MUST be specified if more than one ITS exists at the moment the WDS PCH starts
+  ...
+```
+
+where `name` must specify a name unique among all the control planes in that KubeFlex deployment (note that this must be unique among both ITSes and WDSes), the optional `type` can be either k8s (default) or host, see [here](https://github.com/kubestellar/kubeflex/blob/main/docs/users.md) for more information, the optional `APIGroups` provides a list of APIGroups, see [here](https://docs.kubestellar.io/release-0.22.0/direct/examples/#scenario-2-using-the-hosting-cluster-as-wds-to-deploy-a-custom-resource) for more information, and `ITSName` specify the ITS connected to the new WDS being created (this parameter MUST be specified if more that one ITS exists in the cluster, if no value is specified and only one ITS exists in the cluster, then it will automatically selected).

```

Comment:
s/will automatically/will be automatically/

Reply:
ok, thank you